src/test/java/org/folio/rest/support/CqlUtilTest.java

```diff
@@ -0,0 +1,52 @@
+package org.folio.rest.support;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.folio.rest.testing.UtilityClassTester;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+@RunWith(JUnitParamsRunner.class)
+
+public class CqlUtilTest {
```

Comment:
I'm used to seeing the Annotation next to the Class rather than next to the Import.
I recommend swapping lines 10 and 11.

Reply:
Fixed.